modules/graphics.py

```diff
@@ -0,0 +1,225 @@
+import settings
+import pygame
+from settings import Colors
+import matplotlib
+import matplotlib.pyplot as plt
+import matplotlib.backends.backend_agg as agg
+import pylab
+import numpy as np
+from collections import deque
+from typing import Tuple, List
+
+matplotlib.use("Agg")
+
+
+class Slider:
+    def __init__(self, name, val, maxi, mini, pos):
```

Comment:
`maxi` and `mini` might be specified as a named tuple, since this is a "data clump", perhaps?